workers/proto/kitchen_sink/kitchen_sink.proto

```diff
@@ -0,0 +1,328 @@
+syntax = "proto3";
+
+package temporal.omes.kitchen_sink;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+import "child_workflow.proto";
+import "activity.proto";
+
+import "temporal/api/common/v1/message.proto";
+import "temporal/api/failure/v1/message.proto";
+import "temporal/api/enums/v1/workflow.proto";
+
+// The input to the test overall. A copy of this constitutes everything that is needed to reproduce
+// the test.
+message TestInput {
+  WorkflowInput workflow_input = 1;
+  ClientSequence client_sequence = 2;
+  // Technically worker options should be known as well. We don't have any common format for that
+  // and creating one feels overkill to start with. Requiring the harness to print the config at
+  // startup seems good enough for now.
+}
+
+// All the client actions that will be taken over the course of this test
+message ClientSequence {
+  repeated ClientActionSet action_sets = 1;
+}
+
+// A set of client actions to execute. Client actions are always executed concurrently, since
+// sequential execution can be represented by a series of 1-size action sets.
+message ClientActionSet {
+  repeated ClientAction actions = 1;
+}
+
+message ClientAction {
+  oneof variant {
+    DoSignal do_signal = 1;
+    DoQuery do_query = 2;
+    DoUpdate do_update = 3;
+  }
```

Comment:
Sure, that's resonable

Reply:
Actually, I changed my mind here. It needs to be the way it is in workflows to be able to express certain patterns without a signal in between - that isn't necessary here, and because we have to generate this I'd rather be inconsistent than have a technique that I know I'm just not going to bother using in the generator b/c of the added complexity. The desired patterns are representable without adding the concurrent flag or nested set, so fewer ways for the generator is better.